src/test/java/com/conveyal/datatools/manager/gtfsplus/GtfsPlusValidationTest.java

```diff
@@ -165,4 +173,50 @@ void shouldReportEmptyRows() throws Exception {
             validation.issues.get(0).description, equalTo("1 row(s) are empty. (File may need to be edited manually.)")
         );
     }
+
+    @ParameterizedTest
+    @MethodSource("createIncompleteDirectionsCases")
+    void shouldReportIncompleteDirections(String zipFile, String message) throws Exception {
+        FeedVersion ggfIncompleteDirections = createFeedVersionFromGtfsZip(goldenGateFerry, zipFile);
+
+        LOG.info("Validating GTFS+ directions.txt in Golden Gate Ferry feed should produce errors");
+        GtfsPlusValidation validation = GtfsPlusValidation.validate(ggfIncompleteDirections.id);
+        assertThat(
```

Comment:
Nit. Could replace with assertEquals.

Reply:
I followed the existing tests in this file and other test files, which use `assertThat`. Do you want me to switch the assertions for this test file for this PR?

Reply:
It was only a nit. Leave it as is.